.github/workflows/ci.yml

```diff
@@ -56,5 +57,84 @@ jobs:
 
       - run: mix test --warnings-as-errors --cover
 
+  dialyzer:
+    runs-on: ubuntu-22.04
+    env:
+      MIX_ENV: test
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Elixir
+        id: beam
+        uses: erlef/setup-beam@v1
+        with:
+          elixir-version: 1.16
+          otp-version: 26
+
+      - uses: actions/cache@v4
+        with:
+          path: |
+            deps
+            _build
+          key: deps-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}-${{ hashFiles('**/mix.lock') }}
+          restore-keys: |
+            deps-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}
+
+      - run: mix deps.get
+
+      - name: Restore PLT cache
+        id: plt_cache_restore
+        uses: actions/cache/restore@v3
+        with:
+          key: |
+            plts-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}-${{ hashFiles('**/mix.lock') }}
+          restore-keys: |
+            plts-${{ runner.os }}-${{ steps.beam.outputs.otp-version }}-${{ steps.beam.outputs.elixir-version }}-
+          path: |
+            priv/plts
```

Comment:
Do we need to add an entry to the `dialyzer` block in `mix.exs` specifying that the plts will be in `priv/plts`?

There is a sample of how dialyxir suggests we do it in https://hexdocs.pm/dialyxir/readme.html#continuous-integration

Reply:
hum do we need to ?I did not have to for our CI in podium

Reply:
unresolving this, internally we rely on `plt_local_path` for the same affect, but it needs to be specified in the mix.exs file to put these plts in the `priv/` folder otherwise it will go to `_build` and this cache has no effect.

Reply:
i removed the cache internally because we had issue with caching in the past... and that was causing us having to blow away the cache to pass

Reply:
updated the file but if I see any sort of issue I will remove that again